doc/Design.md

```diff
@@ -464,8 +463,8 @@ The restic backup program guarantees the following:
 
  * Accessing the unencrypted content of stored files and meta data should not
    be possible without a password for the repository. Everything except the
-   `version` and `id` files and the meta data included for informational
-   purposes in the key files is encrypted and authenticated.
+   meta data included for informational purposes in the key files is encrypted and
```

Comment:
I think "meta data" is usually written as "metadata" (https://en.wikipedia.org/wiki/Metadata)